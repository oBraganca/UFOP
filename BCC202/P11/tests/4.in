1 Gabriel 1 25 5 1989
1 Helena 7 11 8 1970
1 Davi 4 11 8 1974
1 Alice 7 19 1 1932
1 Isabella 8 30 10 1943
2
1 Bernardo 6 6 8 1986
1 Gabriel 6 19 12 1999
1 Helena 3 25 10 1952
1 Gabriel 2 29 1 1939
1 Helena 3 9 12 2023
2
1 Sophia 6 8 1 1982
1 Bernardo 6 20 3 1975
1 Alice 10 29 10 2004
1 Bernardo 8 13 4 1955
1 Davi 6 14 1 1970
1 Pedro 9 26 4 2019
1 Bernardo 3 21 4 1934
1 Gabriel 10 5 12 1933
1 Sophia 7 22 11 2022
1 Davi 3 2 5 1970
3
