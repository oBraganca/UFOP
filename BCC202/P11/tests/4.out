08 | 30/10/1943 | Isabella
07 | 19/01/1932 | Alice
10 | 05/12/1933 | Gabriel
10 | 29/10/2004 | Alice
09 | 26/04/2019 | Pedro
08 | 13/04/1955 | Bernardo
07 | 11/08/1970 | Helena
07 | 22/11/2022 | Sophia
06 | 14/01/1970 | Davi
06 | 20/03/1975 | Bernardo
06 | 08/01/1982 | Sophia
06 | 06/08/1986 | Bernardo
06 | 19/12/1999 | Gabriel
04 | 11/08/1974 | Davi
03 | 21/04/1934 | Bernardo
03 | 25/10/1952 | Helena
03 | 02/05/1970 | Davi
03 | 09/12/2023 | Helena
02 | 29/01/1939 | Gabriel
01 | 25/05/1989 | Gabriel
